app/src/main/java/projekt/substratum/fragments/ManagerFragment.java

```diff
@@ -349,10 +349,10 @@ public void onClick(View view) {
     @Override
     public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         inflater.inflate(R.menu.overlays_list_menu, menu);
-        menu.findItem(R.id.restart_systemui).setShowAsAction(MenuItem.SHOW_AS_ACTION_ALWAYS);
-        menu.findItem(R.id.per_app).setShowAsAction(MenuItem.SHOW_AS_ACTION_ALWAYS);
+        menu.findItem(R.id.restart_systemui).setShowAsAction(MenuItem.SHOW_AS_ACTION_IF_ROOM);
```

Comment:
Restart SystemUI is rather mission critical over here so I'm against this.

Reply:
I agree it's important, but surely this can be set in XML.? 
NOTE: Dont kill me for not making a PR! PLEASE

Reply:
It's already set in XML [here](https://github.com/substratum/substratum/blob/dev/app/src/main/res/menu/activity_menu.xml#L28-#L34) but then hidden in code [here](https://github.com/substratum/substratum/blob/dev/app/src/main/java/projekt/substratum/MainActivity.java#L928-#L31) so this is more of a "just make sure it shows up" kind of thing.

Also Jesus Christ dude what is the panic and alarm for? Code review is a normal and welcomed exercise and not everybody who suggests something has to also push a PR to resolve it, as project maintainers it's our own task to deal with it.